string[] detID
bool returnKP
sensor_msgs/Image leftImg
sensor_msgs/Image rightImg
---
front_end/frameDetection[] outputFrames 
